.cm/change_categories.cm

```diff
@@ -121,43 +121,9 @@ platforms:
       - 'subprojects/internal-instrumentation-processor/'
       - 'subprojects/model-core/'
       - 'subprojects/model-groovy/'
-  - core:
-    name: 'core'
+  - core_execution:
+    name: 'core_execution'
     subprojects:
-      - 'subprojects/base-services/'
```

Comment:
🤔 Where did these go? E.g. `wrapper` is still in subprojects, but there is no platform attached to it.

Reply:
I think these changes to `CODEOWNERS` are a mistake and I've messaged Sterling to confirm this.  The commit that changed them looked intentional, though.  For now, I'm just aligning with `CODEOWNERS`, which should be the source of truth.